sklearn/metrics/_plot/roc_curve.py

```diff
@@ -368,18 +368,18 @@ def plot_roc_curve(
     pos_label=None,
     **kwargs,
 ):
-    """Plot Receiver operating characteristic (ROC) curve.
-
-    Extra keyword arguments will be passed to matplotlib's `plot`.
-
-    Read more in the :ref:`User Guide <visualizations>`.
-
-    .. deprecated:: 1.0
+    """.. deprecated:: 1.0
       `plot_roc_curve` is deprecated in 1.0 and will be removed in
        1.2. Use one of the following class methods:
        :func:`~sklearn.metrics.RocCurveDisplay.from_predictions` or
        :func:`~sklearn.metrics.RocCurveDisplay.from_estimator`.
+    
```

Comment:
you need to move these lines as it was previously (see the diff).

Reply:
did as you directed but tests still failing